src/applications/mhv/secure-messaging/tests/components/Compose/FileInput.unit.spec.jsx

```diff
@@ -27,7 +27,9 @@ describe('File input component', () => {
 
   it('should allow a file to be attached', async () => {
     const file = new File(['(⌐□_□)'], 'test.png', { type: 'image/png' });
-    const screen = render(<FileInput attachments={attachments} />);
+    const screen = render(
+      <FileInput attachments={attachments} setAttachments={() => {}} />,
```

Comment:
Was getting a warning/error when calling this without a `setAttachments` function.